data/hai missions.txt

```diff
@@ -1061,3 +1061,366 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+mission "Hai Solifuge response"
+	landing
+	invisible
+	to offer
+		has "Wanderers Hai Assistance 3: done"
+	on offer
+		event "Hai Pond Striders released" 10
+		fail
+
+event "Hai Pond Striders released"
+	fleet "Large Hai"
+		add variant 3
+			"Hai Pond Strider" 2
+			"Flea" 16
+			"Lightning Bug" 2
+		add variant 2
+			"Hai Pond Strider" 3
+			"Flea" 24
+		add variant 1
+			"Hai Pond Strider"
+			"Flea" 8
+			"Geocoris"
+			"Aphid (Anti-Missile)" 2
+			"Grasshopper"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea" 16
+			"Centipede"
+
+mission "Strider: View of the Pond"
+	description "Deliver a message from Ambassador Sayari to Hai Home."
+	name "Message Hai-home"
+	landing
+	source "Vara K'chrai"
+	destination "Hai-home"
+	to offer
+		has "Wanderers Solifuge Recon 1: done"
+	on offer
+		conversation
+			`"Ambassador Sayari has been informed of the Solifuge, has a message for the Hai Homeworld, and needs your help to deliver it to Hai-Home."`
+				accept
+	on complete
+		conversation
+			`The Hai, as usual, mysteriously and efficiently deliver your message from Ambassador Sayari, and a local prefect appears within minutes to inquire if you've encountered a friendly race with hull repair technology.`
+			`The prefect continues to tell you how the Solifuge was not entirely a surprise to the Hai government, but that it posed a threat to the Hai public mindset that couldn't be ignored.`
+
+mission "Strider: Have language, will travel"
+	description "Take Ambassador Yashili and her delegation to <destination>."
+	name "Have language, will travel"
+	landing
+	source "Hai-home"
+	destination "Greenwater"
+	passengers 3
+	blocked "The Hai wish to send a delegation to a species that has hull repair technology, and will require 3 bunks for the trip."
+	to offer
+		has "Strider: View of the Pond: done"
+	on offer
+		log `The Hai are working on a Medium Warship that carries drones, but need help contacting a species with hull repair technology to make it viable.`
+		conversation
+			`"A small, short range drone carrier and combat drone solution is being worked on, but initial simulations predict that actual escort casualties would go up unless significant hull repair technology could be implemented to fix docked drones that would otherwise weigh down the agile warship" the prefect explains.`
+			branch neither
+				not "remnant blood test pure"
+				not "Coalition: First Contact: offered"
+			`The delegation will need to stop at <destination> to pick up translation devices they can use to communicate with other species.`
```

Comment:
wow, an outline that's an even sloppier outline than mine...lol...

i'm going to throw a sentence in that makes the existing format work, mostly because I don't want to spend a whole ton of time writing a new set of missions that basically does the same thing as the ones that are present.

Reply:
I'd be fine with tackling the restructuring of the missions along with the writing myself. In less than two weeks, this college semester will finally be over, so I'll have plenty of time to work on this.